unity-renderer/ProjectSettings/QualitySettings.asset

```diff
@@ -1,51 +1,51 @@
-%YAML 1.1
-%TAG !u! tag:unity3d.com,2011:
---- !u!47 &1
-QualitySettings:
-  m_ObjectHideFlags: 0
-  serializedVersion: 5

```

Comment:
something specific changed in the quality settings or it can be discard for this PR?

Reply:
oops sorry, nice catch